src/AccessibilityInsights.Desktop/UIAutomation/EventHandlers/EventMessage.cs

```diff
@@ -21,7 +21,9 @@ public class EventMessage:IA11yEventMessage, IDisposable
         /// </summary>
         public string TimeStamp { get; set; }
 
-        public List<KeyValuePair<string,dynamic>> Properties { get; internal set; }
```

Comment:
>internal [](start = 68, length = 8)

The setter was changed from public to internal in commit e48552411cf6be6eab31d5122e54f256e1e75161 in the pre-GitHub repo. That commit included the changes to clean up our FxCop violations, and suppressed CA2227 in several places. I've reviewed that commit and this seems to be the only place where the visibility change was chosen over the suppression. This restores the behavior from before that change, while still suppressing the FxCop warning